server/views/partials/additionalLicenceConditionsSummary.njk

```diff
@@ -1,36 +1,29 @@
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
 
 {% macro additionalLicenceConditions(licence, additionalConditions, isEditable, validationErrors, title, hideHeading) %}
     {% set errorMessage = validationErrors | findError('additionalLicenceConditions') %}
+    {% set conditionCount = additionalConditions.length or 0 %}
```

Comment:
length of an empty array would be 0 anyway?

Reply:
If the additionalConditions are undefined then the count is coming as empty.
I found it in one of the test cases.
So I added it for protecting undefined stuff.